content/kots-cli/restore.md

```diff
@@ -0,0 +1,24 @@
+---
+date: 2019-10-30
+linktitle: "restore"
+title: kots restore
+weight: 90220
+---
+
+The `kots restore` restores an instance backup for disaster recovery. 
+
+### Usage
+```bash
+kubectl kots restore [flags]
```

Comment:
Technically, one of these flags, `--from-backup` is required, right? As written, it says all flags are optional. Maybe `--from-backup` should be explicitly listed here, not as a "flag".

Reply:
I should take that back. The [flags] convention doesn't necessarily mean it's optional. Or at least that's how we've been using it elsewhere.

Reply:
i updated the flag to mention that it's required. didn't include it in the command itself though.